src/rdiff_backup/log.py

```diff
@@ -180,7 +187,7 @@ def conn(self, direction, result, req_num):
         else:
             result_repr = str(result)
         # shorten the result to a max size of 720 chars with ellipsis if needed
-        #result_repr = result_repr[:720] + (result_repr[720:] and '[...]')  # noqa: E265
+        # result_repr = result_repr[:720] + (result_repr[720:] and '[...]')  # noqa: E265
```

Comment:
it's ugly but the noqa avoids complaints from flake8, it's my way to make the difference between a true comment and a temporary commenting out of code (I'd like to introduce a way to reduce the amount of data during debug). Just roll-back here. And/or add a TODO marker (would have been cleaner, I admit).

Reply:
Will fix. It's my IDE that reformat it most likely.